src/extras.py

```diff
@@ -37,7 +37,13 @@ def _augment_with_salary_cached(last: Optional[str], first: Optional[str]) -> st
     if not results:
         return ""
     s = results[0]
-    projected = Decimal(s["hourly_rate"]) * 40 * 50
+    hourly_rate = s["hourly_rate"]
+    # City of Seattle wage data formats rate as "$    <number>" for some
+    # horrible reason, so we clean this data a bit for decimal conversion.
+    if isinstance(hourly_rate, str):
+        hourly_rate = hourly_rate.replace("$", "").replace(" ", "")
+    s["hourly_rate"] = hourly_rate
```

Comment:
This is probably meant to be indented one more level and go in the if block.

Reply:
Hmm, it wasn't initially but I guess it doesn't really matter. Either the value is changed and it gets reset, or it's not and it doesn't matter if we reset it :woman_shrugging: